mm2src/coins/utxo/slp.rs

```diff
@@ -1081,6 +1081,8 @@ impl MarketCoinOps for SlpToken {
         Box::new(self.platform_coin.my_balance().map(|res| res.spendable))
     }
 
+    fn base_coin_ticker(&self) -> &str { self.platform_coin.ticker() }
```

Comment:
Maybe the `base_coin_ticker` fn can be renamed to `platform_ticker` while removing the following fn from `slp` impl https://github.com/KomodoPlatform/atomicDEX-API/blob/683d441b70cac2f4435b0388e29f8c4e29025cd3/mm2src/coins/utxo/slp.rs#L757
`platform_ticker` is a better name since the `base_coin` is called `platform` in coin config

Reply:
> Maybe the `base_coin_ticker` fn can be renamed to `platform_ticker` while removing the following fn from `slp` impl
> 
> https://github.com/KomodoPlatform/atomicDEX-API/blob/683d441b70cac2f4435b0388e29f8c4e29025cd3/mm2src/coins/utxo/slp.rs#L757
> 
> 
> `platform_ticker` is a better name since the `base_coin` is called `platform` in coin config

Good catch, and good idea.